webapp/packages/plugin-sql-editor/src/SqlEditor/SqlEditor.tsx

```diff
@@ -101,15 +90,15 @@ export const SqlEditor = observer<ISqlEditorProps>(function SqlEditor({ state, c
       lazy
       onChange={handleModeSelect}
     >
-      <sql-editor className={className}>
-        <SQLEditorActions data={data} state={state} />
-        <TabPanelList />
-        {displayedEditors > 1 ? (
+      {displayedEditors > 1 ? (
```

Comment:
Also I did that to prevent render this AI stuff bug and other similar issues

Reply:
this now totally wrong because we will not render SQL Editor at all when we have only one editor (for example in CE)

this logic was only for hiding tabs when we can't switch to different editor (for example SQL / VQB)